extension/webview/TransactionView.ts

```diff
@@ -25,6 +26,11 @@ export class TransactionView extends ReactView {
     }
 
     async openPanelInner(panel: vscode.WebviewPanel): Promise<void> {
+        panel.webview.onDidReceiveMessage(async (message: any) => {
```

Comment:
In `src/Utils.ts` you call:
```
vscode.postMessage({
    command: message.type,
    data: message.data
})
```

It's worth updating the type of `message: any` to represent that.